packages/common/src/core/schemas/internal/configOptionHelpers.ts

```diff
@@ -150,3 +152,15 @@ async function getGroupsToShareToConfigOptions(
     },
   });
 }
+
+async function getMembershipAccessConfigOptions(
+  _entity: ConfigurableEntity
+): Promise<UiSchemaElementOptions> {
+  const isEditGroup = _entity.isSharedUpdate;
+  return Promise.resolve({
+    scope: "/properties/membershipAccess",
+    options: {
+      disabled: [true, true, isEditGroup],
```

Comment:
should this be `[false, false, isEditGroup`]?

Reply:
Thanks for catching that, updated.